user-guide/Cloud_Platform/RemoteAccess/Cloud_Remote_Access.md

```diff
@@ -16,9 +16,17 @@ With this feature, users can:
 
 - Access files or webpages on the DMA that have been made available for remote access.<!-- RN 38426 -->
 
-  To access such files, use the remote access URL followed by `/public/` (e.g. the file *image.png* via `https://ziine-skyline.on.dataminer.services/public/image.png`).
+  The files or webpages that are available for public access are located in the following folders on the DMA:
 
-  To make files or webpages available for remote access, add them in the folder `C:\Skyline DataMiner\Webpages\public\` on the DMA.
-  
-  > [!NOTE]
-  > Files in the folder `C:\Skyline DataMiner\Webpages\public\` will be hosted without authentication on the internal network of a DMA. However, only authenticated members of the DMS will be able to access them via remote access.
+  - `C:\Skyline DataMiner\Webpages\public\`
+
+    To access these files, use the remote access URL followed by `/public/` (e.g. the file *image.png* via `https://ziine-skyline.on.dataminer.services/public/image.png`).
+
+    > [!NOTE]
+    > Files in the folder `C:\Skyline DataMiner\Webpages\public\` will be hosted without authentication on the internal network of a DMA. However, only authenticated members of the DMS will be able to access them via remote access.
```

Comment:
"only authenticated members of the DMS will be able to acces ..."
It's not really the members of the local DMS, but rather members of the organization and dms/coordination configured on admin.dataminer.services.
Can be that this is clear enough, just wanted to point it out

Reply:
I'm not sure if it's necessary to add this, as it could make things needlessly complex. Can a cloud user be a member of a DMS without also being a member of the corresponding organization?

Reply:
Can be left, no need to make it more complex.

"Can a cloud user be a member of a DMS without also being a member of the corresponding organization?"
No, he/she needs access on both the organization and dms on admin.dataminer.services